src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -223,4 +230,14 @@ private async Task TryCleanupActivityV3(IDurableOrchestrationContext context, IR
         }
         catch (Exception) { }
     }
+
+    private string GetSerializedInput(UpdateCheckpoint checkpoint) => JsonSerializer.Serialize(new
```

Comment:
So we're re-using the `Input` column in the diagnostic table?

Reply:
Yes we log while we create the operation, I want to log when we complete the operator or when we error